readme.md

```diff
@@ -44,7 +19,7 @@ Use it to:
 
 You plug it into your app as a dev dependency so it adds nothing to your product builds.
 
-The app comes in two forms, which share the same [back end](https://github.com/reactotron/reactotron/tree/master/packages/reactotron-core-server):
+The app comes in two forms which share the same [guts](https://github.com/reactotron/reactotron/tree/master/packages/reactotron-core-client):
```

Comment:
Don't be sad, you made me realize I should document how things are put together!  

I created a placeholder called "What's Inside This Repo?".  It'll be my tribute to you.